app/templates/main/how-to-use-this-service.html

```diff
@@ -0,0 +1,15 @@
+{% extends "base.html" %}
+{%- from 'govuk_frontend_jinja/components/inset-text/macro.html' import govukInsetText -%}
```

Comment:
This import isn't being used, could we remove it?

Reply:
done